tests/userLogin.test.js

```diff
@@ -0,0 +1,29 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import { startServer } from '../src/server';
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+
+describe('Users Login', () => {
+  let app = null;
+  let agent = null;
+
+  before(async () => {
+    app = await startServer(5000);
+    agent = chai.request(app);
+  });
+
+  it('should login user', (done) => {
+    agent.post('/api/v1/login')
+      .send({
+        email: 'martins@gmail.com',
+        password: '%RYYT&^UTB*UYT*IUYIU',
+      })
+      .end((err, res) => {
+        expect(res).to.have.status(200);
```

Comment:
Sync with @chikeozulumba for better testing.

Reply:
I have written more test. Thank you.